### PR TITLE
chore: release 13.7.0

### PR DESCRIPTION
**Pull request by cacieprins**

### CODE REVIEW

File: docs/api/commands/screenshot.mdx [hunk 1]
```diff
@@ -300,6 +300,17 @@ cy.screenshot()
 cy.get('.sticky-header').invoke('css', 'position', null)
 ```
 
+### Chromium-specific behavior with regard to tabs
+
+Chromium will not capture screenshots when the Renderer process for the Cypress
```
Comment (AtofStryker) [inline docs/api/commands/screenshot.mdx:305]: Is this supposed to be lower case? I'm not actually sure 😬 
```suggestion
Chromium will not capture screenshots when the renderer process for the Cypress
```

File: docs/guides/references/changelog.mdx [hunk 2]
```diff
@@ -2,6 +2,37 @@
 title: Changelog
 ---
 
+## 13.7.0
+
+**Features:**
+
+- Added shadow DOM snapshot support within Test Replay in order to highlight elements correctly within the Cypress reporter. Addressed in [#28823](https://github.com/cypress-io/cypress/pull/28823).
+- Added TypeScript support for [Vue 2.7+](https://github.com/vuejs/vue/blob/main/CHANGELOG.md#270-2022-07-01). Addresses [#28591](https://github.com/cypress-io/cypress/issues/28591).
+- Adds additional context to error messages displayed when Test Replay artifacts fail to upload. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986)
```
Comment (AtofStryker) [inline docs/guides/references/changelog.mdx:11]:
```suggestion
- Adds additional context to error messages displayed when Test Replay artifacts fail to upload. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986).
```

File: docs/guides/references/changelog.mdx [hunk 3]
```diff
@@ -2,6 +2,37 @@
...
+
+**Performance:**
+
+- Fixed a performance regression from [`13.6.3`](/guides/references/changelog#13-6-3) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
+- Reduces the number of attempts to retry failed Test Replay artifact uploads from 8 to 3, to reduce time spent on artifact upload attempts that will not succeed. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986)
```
Comment (AtofStryker) [inline docs/guides/references/changelog.mdx:16]:
```suggestion
- Reduces the number of attempts to retry failed Test Replay artifact uploads from 8 to 3, to reduce time spent on artifact upload attempts that will not succeed. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986).
```

File: docs/guides/references/changelog.mdx [hunk 4]
```diff
@@ -2,6 +2,37 @@
...
+
+**Bugfixes:**
+
+- Changed screenshot capture behavior in Chromium to activate the main Cypress tab before capturing. This prevents screenshot capture from timing out in certain situations. Fixed in [#29038](https://github.com/cypress-io/cypress/pull/29038). Fixes [#5016](https://github.com/cypress-io/cypress/issues/5016)
+- Fixed an issue where `.click()` commands on children of disabled elements would still produce "click" events -- even without `{ force: true }`. Fixes [#28788](https://github.com/cypress-io/cypress/issues/28788).
+- Changed RequestBody type to allow for boolean and null literals to be passed as body values. [#28789](https://github.com/cypress-io/cypress/issues/28789)
```
Comment (AtofStryker) [inline docs/guides/references/changelog.mdx:22]:
```suggestion
- Changed RequestBody type to allow for boolean and null literals to be passed as body values. [#28789](https://github.com/cypress-io/cypress/issues/28789).
```

File: docs/guides/references/changelog.mdx [hunk 5]
```diff
@@ -2,6 +2,37 @@
...
+
+**Misc:**
+
+- Changed Component Testing scaffolding instruction to `pnpm add` to add framework dependencies when a project uses pnpm as package manager. Addresses [#29052](https://github.com/cypress-io/cypress/issues/29052).
```
Comment (AtofStryker) [inline docs/guides/references/changelog.mdx:26]:
```suggestion
- Changed Component Testing scaffolding instruction to `pnpm add` to add framework dependencies when a project uses `pnpm` as package manager. Addresses [#29052](https://github.com/cypress-io/cypress/issues/29052).
```

File: docs/guides/references/changelog.mdx [hunk 6]
```diff
@@ -2,6 +2,37 @@
...
+- Command messages in the Cypress command logs will now truncate display at 100 lines instead of 50. Fixes [#29023](https://github.com/cypress-io/cypress/issues/29023).
+- Capture the `beforeTest` timestamp inside the browser for the purposes of accurately determining test start for Test Replay. Addressed in [#29061](https://github.com/cypress-io/cypress/pull/29061).
+
+**Dependency Updates:**
+
+- Updated jimp from `0.14.0` to `0.22.12`. Addressed in [#29055](https://github.com/cypress-io/cypress/pull/29055).
```
Comment (AtofStryker) [inline docs/guides/references/changelog.mdx:32]: we've linked the dependency libraries in the past when we do these updates. Should we do the same here?

File: docs/guides/references/changelog.mdx [hunk 7]
```diff
@@ -2,6 +2,37 @@
 title: Changelog
 ---
 
+## 13.7.0
+
+**Features:**
```
Comment (MikeMcC399) [inline docs/guides/references/changelog.mdx:7]: The release date is missing.

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**